tendrl_post.yml

```diff
@@ -0,0 +1,17 @@
+---
+#
+# Tendrl server
+#
+
+- hosts: tendrl-server
+  remote_user: root
+  tasks:
+    - name: Read Tendrl admin password
+      command: cat /root/password
+      register: admin_password
+
+    - debug: var=hostvars[groups['tendrl-server'][0]]['admin_password']
```

Comment:
If I remember it corectly, it is because the next tasks are "local" tasks, so you have to specify from which machine you need the variable. (Not in this case, but theoretically there might be more than one machine in `tendrl-server` group.)